fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlAdapter.java

```diff
@@ -302,4 +304,33 @@ public void createIndex(String schemaName, String tableName, String indexName, S
         String ddl = DataDefinitionUtil.createIndex(schemaName, tableName, indexName, indexColumns, false);
         runStatement(ddl);
     }
-}
+
+    @Override
+    public boolean checkCompatibility(String adminSchema) {
+        final String statement = "SELECT 1";
+        boolean result = false;
+        try (Connection c = connectionProvider.getConnection(); PreparedStatement stmt = c.prepareStatement(statement)) {
```

Comment:
does this always return true?
the comments for --checkCompatibility "checks feature compatibility" seems a little bit confusing for me...
I didn't notice this before, could you help to tell what's this used for?

Reply:
it's used to check the connectivity to the database. 
returns an exception if there is an issue.

Reply:
ha, interesting,  the function name seems a little bit confusing, I though it's compatible with something ...